src/Nethermind/Nethermind.Merge.Plugin/MergePlugin.cs

```diff
@@ -66,8 +66,14 @@ public partial class MergePlugin : IConsensusWrapperPlugin, ISynchronizationPlug
         public string Description => "Merge plugin for ETH1-ETH2";
         public string Author => "Nethermind";
 
-        public virtual bool MergeEnabled => _mergeConfig.Enabled;
+        public virtual bool MergeEnabled => _mergeConfig.Enabled &&
+                                            !IsPreMergeConsensusAuRa(_api); // AuRa has dedicated plugin AuRaMergePlugin
```

Comment:
I think we should have all the AuRa checks in `AuRaMergePlugin` and we can falsify `_mergeConfig.Enabled` from that plugin. But there is no need to have awareness of AuRa consensus in `MergePlugin` code

Reply:
and that is why you needed AuraMerge.Enabled. If you disable _mergeConfig.Enabled without relaying on AuRa check we have to store mergeAuraEnabled in some variable. IMO, both approaches are not perfect, but I prefer my approach